app/src/main/java/com/cornellappdev/android/eatery/ui/components/general/EateryCard.kt

```diff
@@ -365,59 +323,68 @@ fun EateryCardPrimaryHeader(eatery: Eatery, isCompact: Boolean) {
             DotSeparator()
             EateryMenuSummary(eatery = eatery)
         }
-    } else {
+    }
+}
+
+@Composable
+fun EateryCardSecondaryHeader(eatery: Eatery, style: EateryCardStyle = EateryCardStyle.DEFAULT) {
+    if (style != EateryCardStyle.COMPACT) {
         Row(
-            modifier = Modifier.padding(top = 2.dp)
+            modifier = Modifier.padding(top = 2.dp),
+            verticalAlignment = Alignment.CenterVertically
         ) {
+            val openUntil = eatery.getOpenUntil()
             Text(
-                text = eatery.location ?: "Unknown location",
-                color = GrayFive,
-                style = EateryBlueTypography.subtitle2
+                modifier = Modifier.padding(top = 2.dp),
+                text =
+                if (openUntil == null) "Closed"
+                else if (eatery.isClosingSoon()) "Closing at $openUntil"
+                else ("Open until $openUntil"),
+                style = EateryBlueTypography.subtitle2,
+                color = if (openUntil == null) Red
+                else if (eatery.isClosingSoon()) Yellow
+                else Green
             )
-            EateryMenuSummary(eatery = eatery)
+
         }
     }
 }
 
+//TODO, integrate backend for the entree recommendation once backend is done for that
 @Composable
-fun EateryCardSecondaryHeader(eatery: Eatery, isCompact: Boolean) {
-    if (!isCompact && eatery.getWalkTimes() != null) {
-        val walkText =
-            "${if (eatery.getWalkTimes()!! > 0) eatery.getWalkTimes() else "< 1"} min walk"
```

Comment:
Hmm yeah seems like we deleted walk times? This is a pretty useful metric so we definitely wanna keep this info.

Right now the figma shows wait times but those don't exist, so you can use the walk icon and display walk times in the same spot:

![image](https://github.com/user-attachments/assets/80bd3316-4a6b-4d9f-adae-23605ed8031c)

Reply:
Fixed the UI for eatery cards, see attached images below. A Eatery card now have no additional space between the Eatery Title and primary header, the primary header now displays the location, and the secondary header displays walk times for list view cards
<img width="312" alt="Screenshot 2024-12-06 at 1 17 31 PM" src="https://github.com/user-attachments/assets/1c90044b-1c19-4c4c-b990-f47d7d7b7259">
<img width="311" alt="Screenshot 2024-12-06 at 1 19 23 PM" src="https://github.com/user-attachments/assets/38ad436a-bf41-4036-b215-ce3e9afa6894">